tokio/tests/sync_mutex.rs

```diff
@@ -147,3 +147,11 @@ fn try_lock() {
     let g3 = m.try_lock();
     assert_eq!(g3.is_ok(), true);
 }
+
+#[tokio::main]
+#[test]
```

Comment:
Both of these lines can become `#[tokio::test]`

Reply:
Thanks for your review. I have corrected it and other inconsistencies in the same file.